.changeset/small-lions-fold.md

```diff
@@ -0,0 +1,7 @@
+---
+'braid-design-system': patch
+'@braid-design-system/codemod': patch
```

Comment:
The codemod doesn't have a peer dep on `react`. It still needs a release, but only because of dependency updates.